src/dataexplorer/dataexplorer-application/src/components/SchemaResponse/Schemes/Schemes.tsx

```diff
@@ -0,0 +1,281 @@
+                                                        {qa.answers?.answers.answer?.map(
+                                                            (item) => (
+                                                                <p
+                                                                    className="answers"
+                                                                    key={
+                                                                        item
+                                                                            .valueCoding
+                                                                            ?.display
+                                                                            ? qa
+                                                                                  .answers
+                                                                                  ?.id +
+                                                                              item
+                                                                                  .valueCoding
+                                                                                  .display
+                                                                            : qa
+                                                                                  .answers
+                                                                                  ?.id
+                                                                    }
+                                                                >
+                                                                    {
+                                                                        item.valueBoolean
+                                                                    }
+                                                                    {
+                                                                        item
+                                                                            .valueCoding
+                                                                            ?.display
+                                                                    }
+                                                                    {
+                                                                        item.valueDate
+                                                                    }
+                                                                    {
+                                                                        item.valueDecimal
+                                                                    }
+                                                                    {
+                                                                        item.valueString
+                                                                    }
+                                                                </p>
```

Comment:
Is this the best formatting for something like this? I dont know what the best practices are, but it just looks like it could be presented on way fewer lines, instead of using 3 lines to present each item. I don't know. Thoughts?

Reply:
Agree, but prettier format's it this way's. Only way to avoid it is to change the default prettier rules for all files

Reply:
I guess it doesn't matter too much, we can look into prettifying the code at a later time.